bibliopixel/drivers/spi_driver_base.py

```diff
@@ -50,11 +50,12 @@ def __init__(self, **kwargs):
 
         try:
             from periphery import SPI
-            self._spi = SPI(self._dev, 0, self._spi_speed*1e6)
+            self._spi = SPI(self._dev, 0, self._spi_speed * 1e6)
         except ImportError:
             error(CANT_IMPORT_PERIPHERY_ERROR)
 
-        log.info('periphery spi dev {:s} speed @ {:.2f} MHz'.format(self._dev, self._spi.max_speed/1e6))
+        log.info('periphery spi dev {:s} speed @ {:.2f} MHz'.format(
```

Comment:
I still plan on a separate commit where I rework the SPI stuff... for one, it's unusable in projects not because the ```interface``` param takes a class! That ain't gonna work :P So I have to make that an enumeration or something. For the record, I do the same thing all the time... it's easier, but just incompatible with projects... that commit is next after this.